pyproject.toml

```diff
@@ -25,38 +25,34 @@ classifiers = [
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Unix',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
 ]
 
 [tool.poetry.dependencies]
-# <3.10 being dictated currently by smqtk-detection, which could use some TLC to open up.
-python = ">=3.7,<3.10"
-numpy = [
-    { version = ">=1.21.1", python = "<3.8" },
-    { version = ">=1.23.5", python = ">=3.8" }
-]
-scikit-image = ">=0.18.1"
-scikit-learn = [
-    { version = ">=0.24.2", python = "<3.8" },
-    { version = ">=1.2", python = ">=3.8" }
+python = "^3.8"
+numpy = ">=1.22"
+scikit-image = [
+    # Hinge because minimum support 0.20.0 for py3.11
+    { version = ">=0.18.1", python = "<3.11" },
+    { version = ">=0.20.0", python = ">=3.11" }
 ]
+scikit-learn = ">=1.2"
 smqtk-classifier = ">=0.17.0"
 smqtk-core = ">=0.18.0"
 smqtk-descriptors = ">=0.16.0"
-smqtk-detection = ">=0.19.0"
-scipy = [
-    { version = ">=1.6.3", python = "<3.8" },
-    { version = ">=1.8.1", python = ">=3.8" }
-]
+smqtk-detection = { git = "https://github.com/Kitware/SMQTK-Detection.git", branch = "dev/update-to-3.8-minimum" }
+scipy = ">=1.8.1"
 click = ">=8.0.3"
+setuptools = "*"
 # Optionals for "example" extra
 jupyter = { version = ">=1.0.0", optional = true }
 matplotlib = { version=">=3.4.1", optional = true }
 papermill = { version = ">=2.3.3", optional = true }
-torch = { version = ">=1.9.0", optional = true }
-torchvision = { version = ">=0.10.0", optional = true }
+torch = {version = ">=1.13.1,!=2.0.1", optional = true}
```

Comment:
To note, the dependabot warning that motivated this is more specifically pertaining to our *concrete* versioning listed in the `poetry.lock` file, not here. We discussed at an earlier time that we may encounter incompatibilities with other packages because we claim to not work with versions below 1.13.1, but we actually can. The "not recommended" part (dependabot) is external to if someone does. E.g. We could keep the lower bound on the abstract requirement at `1.9.0` while maintaining our use of >=1.13.1 in our concrete requirement like it currently is (set to `2.0.0`).